test/go-chaincode/build.bats

```diff
@@ -1,5 +1,31 @@
 #!/usr/bin/env bats
 
-@test "Placeholder golang build test script" {
+setup() {
+  src_dir="scripts"
+}
+
+@test "build.sh: should exist and be executable" {
+  [ -x "${src_dir}/go-chaincode/build.sh" ]
+}
+
+@test "build.sh: should install fabric STUB" {
+  skip
+}
+
+@test "build.sh: should install go STUB" {
   skip
 }
+
+@test "build.sh: should not overwrite existing chaincode src dir contents" {
```

Comment:
Not sure I understand the value of this test. There's not much logic in the build script at the moment, so it's probably enough to test that it calls `go build` with the expected parameters. We should be unit testing the build-lib scripts here rather than the go build, so using a stub should keep things simple and make test_container.sh redundant.